scripts/opentelemetry/appsignals_custom_otel_setup/custom-opentelemetry.yaml

```diff
@@ -0,0 +1,591 @@
+            label_value: ""
+            new_value: ""
+          - action: add_label
+            new_label: Telemetry.Source
+            new_value: RuntimeMetric
+            aggregation_type: ""
+            experimental_scale: 0
+            label: ""
+            label_value: ""
+        - include: jvm.gc.collections.count
+          action: insert
+          new_name: JVMGCYoungGenCount
+          match_type: strict
+          experimental_match_labels: {"name": "G1 Young Generation"}
+          aggregation_type: ""
+          submatch_case: ""
+          operations:
+          - action: aggregate_labels
+            label_set: []
+            aggregation_type: sum
+            experimental_scale: 0
+            label: ""
+            new_label: ""
+            label_value: ""
+            new_value: ""
+          - action: add_label
+            new_label: Telemetry.Source
+            new_value: RuntimeMetric
+            aggregation_type: ""
+            experimental_scale: 0
+            label: ""
+            label_value: ""
+
+    exporters:
+      otlphttp:
+        #traces_endpoint field follows this pattern https://xray.<region>.amazonaws.com/v1/traces
+        traces_endpoint: https://xray.us-east-1.amazonaws.com/v1/traces
+        auth:
+          authenticator: sigv4auth
+
+      awsemf:
+        region: 'us-east-1'
+        log_group_name: "/aws/application-signals/custom"
+        namespace: "ApplicationSignalsCustomPoM"
+        dimension_rollup_option: "NoDimensionRollup"    
+
+      awsemf/application_signals:
+        region: 'us-east-1'
+        log_group_name: "/aws/application-signals/data"
+        namespace: "ApplicationSignals"
+        dimension_rollup_option: "NoDimensionRollup"
+        version: "1"
+        local_mode: true
+        metric_declarations:
+          - dimensions:
+              - [ Environment, Service ]
+            label_matchers:
+              - label_names:
+                  - Telemetry.Source
+                regex: '^RuntimeMetric$'
+                separator: ;
+            metric_name_selectors:
+              - '^.*$'
+
+    service:
+      pipelines:
+        traces/otel:
+          receivers: [otlp]
+          processors: [awsapplicationsignals]
+          exporters: [otlphttp]
+        metrics:
+          receivers: [otlp]
+          processors: 
+          - metricstransform/application_signals
+          - awsapplicationsignals
+          - resourcedetection
+          exporters: [awsemf/application_signals]
+        metrics/2:
+          receivers: [otlp]
+          exporters: [awsemf]
+      extensions: [sigv4auth]
+      telemetry:
+        logs:
+          level: debug
+---
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: my-instrumentation
+spec:
+  # OTEL_EXPORTER_OTLP_ENDPOINT
+  exporter:
+    endpoint: http://applicationsignals-otel-collector:4318
+  nodejs:
+    image: public.ecr.aws/n7z4r8f9/adot-js-amznlinux:latest
```

Comment:
This can be changed to the Official Node.js image which now supports the `copy -r` command:
`public.ecr.aws/aws-observability/adot-autoinstrumentation-node:v0.3.0`